sporkfish/searcher/negamax.py

```diff
@@ -39,49 +41,75 @@ def _negamax(
         """
         value = -float("inf")
 
```

Comment:
It is my fault tbh, but this function is way to big (>100 lines long), very hard to read...

Happy to refactor it later myself...(refactoring [sparks joy](https://konmari.com/marie-kondo-rules-of-tidying-sparks-joy/) for me tbf)

According to [Uncle Bob](https://softwareengineering.stackexchange.com/questions/133404/what-is-the-ideal-length-of-a-method-for-you#:~:text=In%20Clean%20Code%3A%20A%20Handbook,not%20be%20100%20lines%20long.) - functions should rarely by more than 20 lines (esp when we are already writing in python lol)

Reply:
I suspect it wont be as easy, but please do if it sparks joy :)